integration-tests/src/test/resources/environment-configuration-docker.yml

```diff
@@ -58,3 +58,8 @@ zosmfServiceConfiguration:
     serviceId: mockzosmf
 auxiliaryUserList:
     value: 'unauthorized,USER1,validPassword;servicesinfo-authorized,USER,validPassword;servicesinfo-unauthorized,USER1,validPassword'
+idpConfiguration:
+    host: https://dev-95727686.okta.com
+    clientId: 0oa6a48mniXAqEMrx5d7
```

Comment:
this should go to secrets, I think there is one already

Reply:
The clientId is now read from the Github secrets or the environment variable when running the test locally.